CoreScriptsRoot/Modules/Server/ServerChat/DefaultChatModules/PrivateMessaging.lua

```diff
@@ -49,23 +48,23 @@ local function Run(ChatService)
 
 		end
 	end
-	
+
 	local function WhisperCommandsFunction(fromSpeaker, message, channel)
 		local processedCommand = false
 
 		if (string.sub(message, 1, 3):lower() == "/w ") then
```

Comment:
this right here would be a use case of the "message command token parsing" library ;)